UnityProject/Assets/Scripts/Editor/Addressable/AddressableReferencePropertyDrawer.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using AddressableReferences;
+using UnityEditor;
+using UnityEditor.AddressableAssets;
```

Comment:
Codacy found an issue: [Change this constant to a 'static' read-only property.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)